website/docs/d/network_routed_v2.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_network_routed_v2"
+sidebar_current: "docs-vcd-data-source-network-routed-v2"
+description: |-
+  Provides a vCloud Director Org VDC routed Network. This can be used to reference internal networks for vApps to connect.
+---
+
+# vcd\_network\_routed\_v2
+
+Provides a VMware Cloud Director Org VDC routed Network data source. This can be used to reference internal networks for vApps to connect.
```

Comment:
> This can be used to reference internal networks for vApps to connect.

Please rephrase this sentence. When reading for the first time it feels as if we're talking about vApp networks :)

Reply:
Indeed I have copied this from first iteration but it somewhat doesn't read well. Changed to second attempt.